engine/src/main/java/org/pentaho/di/www/GetJobStatusServlet.java

```diff
@@ -195,6 +195,8 @@ public void doGet( HttpServletRequest request, HttpServletResponse response ) th
 
     String jobName = request.getParameter( "name" );
     String id = request.getParameter( "id" );
+    String root = request.getRequestURI() == null ? StatusServletUtils.PENTAHO_ROOT : request.getRequestURI().substring( 0, request.getRequestURI().indexOf( CONTEXT_PATH ) );
```

Comment:
line is past 80 line character limit. same for others

Reply:
@NJtwentyone There are dozens of lines in the servlets for carte that are longer than 80...for some reason, wingman/checkstyle doesn't care

Reply:
It still needs to be human readable. I would only worry about the actually variable assigned lines (ie all code besides the "out.println(..." ) that this PR changed. The numerous out.println statements makes sense to break the 80 line limit to make the xml readable.

Reply:
for the ternary operators I was looking for something like this
`
String root = request.getRequestURI() == null
  ? StatusServletUtils.PENTAHO_ROOT
  : request.getRequestURI().substring( 0, request.getRequestURI() 
     .indexOf( CONTEXT_PATH ) );
`
It's not a big deal now, there are other issues with the code style for this file as you mentioned.